README.md

```diff
@@ -219,14 +271,14 @@ If you are using a remote database, no changes are needed.
 Set the new image version in an environment variable:
 
 ```shell
-export IMAGE_CONJUR=gcr.io/cloud-marketplace/cyberark/conjur-open-source:1.0
+export NEW_VERSION=mytest
```

Comment:
This seems like an iffy variable if it's used only once and isn't a semver style

Reply:
Switched this to a semver version number.